themes/bootstrap3/templates/search/facet-list-content.phtml

```diff
@@ -0,0 +1,21 @@
+<ul class="full-facet-list facet__list facet-group" id="facet-list-<?=$this->escapeHtmlAttr($key) ?>">
+  <?php if ($this->page > 1 && !$this->inLightbox): ?>
+    <a href="<?=$this->escapeHtmlAttr($urlBase . '&' . http_build_query(['facetpage' => $this->page - 1, 'facetsort' => $this->sort, 'contains' => $this->contains]))?>" class="facet js-facet-prev-page" data-page="<?=($this->page + 1) ?>" data-sort="<?=$this->escapeHtmlAttr($this->sort) ?>" data-limit="<?=count($this->data) ?>"><?=$this->transEsc('prev_ellipsis') ?></a>
```

Comment:
These paging link lines are still really long. I'd suggest using the htmlAttributes helper to build the attributes first. This would have the added benefit that you could just modify the same attributes for the next page link. That would ensure they stay in sync in future too.

Reply:
Looks good to me. I notice that before the refactoring, the next page data-sort value was set to `$key` rather than `$this->sort`, which seems like it may have been an error that is now corrected.

Reply:
I'll wait for @EreMaijala's confirmation, and then I'll run the test suite one last time and see if @sturkel89 has time for a final review.

Reply:
Right, I missed some of the differences. It's all clear now, and I think the end result is so much easier to understand now. Thanks!